src/slime_parser.peg.eex

```diff
@@ -8,10 +8,10 @@ doctype <- space? 'doctype' space (!eol .)+ eol;
 % Also remove dedent from eol definition and modify transforms accordingly
 tags_list <- tag (crlf indent? empty_lines:(space? crlf indent?)* tag:tag dedent*)*;
 
-% NOTE: We have to track leading spaces for verbatim_text, so it handled separately
-tag <- verbatim_text / space? tag_item / blank:(space? &eol);
+% NOTE: Leading whitespace is handled separately for text items
+tag <- text_item / space? tag_item / blank:(space? &eol);
```

Comment:
I think it is better to keep `verbatim_text` separated from `comment`, so here it would be `comment / verbatim_text / ...`